src/JWT.ts

```diff
@@ -351,7 +446,7 @@ function verifyJWSDecoded(
  */
 export function verifyJWS(jws: string, pubKeys: VerificationMethod | VerificationMethod[]): VerificationMethod {
   const jwsDecoded: JWSDecoded = decodeJWS(jws)
-  return verifyJWSDecoded(jwsDecoded, pubKeys)
+  return verifyJWSDecoded(jwsDecoded as unknown as JWTDecoded, pubKeys)
```

Comment:
This seems unsafe.
`verifyJWS` is an exported function. Its users don't expect the payload to be decodable.

Reply:
I have removed this typing